app/controllers/external_api/v1/permit_applications_controller.rb

```diff
@@ -47,6 +47,10 @@ def show_integration_mapping
 
   private
 
+  def set_permit_application
+    @jurisdiction = Jurisdiction.find(current_external_api_key.jurisdiction_id)
+  end
```

Comment:
this is conflicting now with the method below - not sure which one is correct but should only have one